Header header
int32 image_width
int32 image_height
geometry_msgs/Point[] points
